selfdrive/sensord/tests/test_sensord.py

```diff
@@ -2,12 +2,15 @@
 
 import time
 import unittest
+import numpy as np
+from collections import namedtuple
 
 import cereal.messaging as messaging
+from cereal import log
 from system.hardware import TICI
 from selfdrive.test.helpers import with_processes
-
-TEST_TIMESPAN = 10
+from selfdrive.manager.process_config import managed_processes
+from smbus2 import SMBus
```

Comment:
add this to the top group of imports, import convention is stdlib + third party then openpilot imports